app/Routes.php

```diff
@@ -375,7 +375,7 @@ function ( string $value ) {
 				try {
 					$amount = Euro::newFromFloat(
 						( new AmountParser( 'en_EN' ) )->parseAsFloat(
-							$request->get( 'betrag_auswahl', $request->get( 'amountGiven', '' ) )
+							$request->get( 'betrag_auswahl', $request->get( 'betrag', 'amountGiven' ) )
```

Comment:
This will likely fail, because if `betrag` is empty, the the returned value will not be the value of the `amountGiven` parameter, but the literal string `amountGiven`. To check a sequence of parameters you have to nest the calls to `$request->get`, each call being the default for the next one and the last default an empty string.